pkg/apis/scheduling/v1beta1/labels.go

```diff
@@ -71,3 +71,11 @@ const TopologyDecisionAnnotation = "volcano.sh/topology-decision"
 
 // PodQosLevel is the key of pod qos level
 const PodQosLevel = "volcano.sh/qos-level"
+
+// NetworkTopologyModeAnnotationKey is the annotation key for network topology mode
+// Valid values are defined by the NetworkTopologyMode type (e.g. HardNetworkTopologyMode).
+const NetworkTopologyModeAnnotationKey = "volcano.sh/network-topology-mode"
+
+// NetworkTopologyHighestTierAnnotationKey is the annotation key for network topology highest tier
+// Value should be an integer representing the highest tier allowed
+const NetworkTopologyHighestTierAnnotationKey = "volcano.sh/network-topology-highest-tier"
```

Comment:
I'm thinking about whether to separate all fields for configuration or use a single key with a struct as the value. Because there might be more fields in the future, especially since task-level topology configuration might be added in the future

Reply:
I strongly recommend using a struct: it's more extensible, structurally clear, and well-suited for adding new fields in the future.

Reply:
It's ok to seperate them.